src/main/java/org/opentripplanner/updater/vehicle_sharing/parking_zones/ParkingZonesUpdater.java

```diff
@@ -45,7 +49,8 @@ private DropoffVehicleEdge getRentVehicleAnywhereEdge(Vertex v) {
     @Override
     protected void runPolling() {
         LOG.info("Polling parking zones from API");
-        List<GeometryParkingZone> geometryParkingZones = parkingZonesGetter.getParkingZones(url, graph);
+        List<GeometryParkingZone> geometryParkingZones = parkingZonesGetter.getFromHasura(graph, url, new TypeToken<ApiResponse<ParkingZone>>() {
+        }.getType());
```

Comment:
Stworzyłem nowy branch w którym usunałem dependencję z GSON na rzecz Jacksona i poprawiłem trochę API by nie definiować tego typu za każdym razem jak jak wołamy `getFromHasura`. To jest trochę wada generyków Javy i w teori da się to obejść refleksją ale chyba nie warto.

Patrz na branch `hasura-client-rework-api`

Reply:
Jakbyś mógł sobie cherrypickować te zmiany to byłoby super

Reply:
Z review czekam na dalszy rozwój wydarzeń

Reply:
Dodałem Twoje zmiany